lib/active_model/serializer.rb

```diff
@@ -151,7 +151,7 @@ def success?
     #     serializer.as_json(include: { posts: { include: { comments: { only: :body } }, only: :title } })
     def serializable_hash(adapter_opts = nil)
       adapter_opts ||= {}
-      adapter_opts = { include: '*', adapter: :attributes }.merge!(adapter_opts)
+      adapter_opts = { include: config.default_includes, adapter: :attributes }.merge!(adapter_opts)
```

Comment:
but should be `ActiveModelSerializer.default_includes`, not `config.default_includes`